docs/discover/search.asciidoc

```diff
@@ -88,6 +88,67 @@ If the saved search is associated with a different index pattern than is current
 selected, opening the saved search changes the selected index pattern. The query language
 used for the saved search will also be automatically selected.
 
+[[save-load-delete-query]]
+=== Saving and Loading Queries
+Saving queries enables you to reload them into <<discover, Discover>>, <<visualize, visualizations>> and <<dashboard, dashboards>>.
+Saving a query saves the query string, the filters and the time filter but, unlike searches, the index pattern is not saved, making saved queries portable.
+
+==== Read only access
+When you have insufficient privileges to save queries, the following indicator in Kibana will be displayed and the *Save* button won't be visible in the saved query management pop-over.
```

Comment:
There's no indicator in this screenshot so I think you can remove that from the text:

```suggestion
When you have insufficient privileges to save queries the *Save* button won't be visible in the saved query management popover.
```

Reply:
https://github.com/elastic/kibana/pull/43731/commits/b3163d0499ebb516e91879a34d4508e9e5deca57